src/System.Private.ServiceModel/src/System.Private.ServiceModel.builds

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Private.ServiceModel.csproj" />
+    <Project Include="System.Private.ServiceModel.csproj">
```

Comment:
@weshaggard -- is it correct to assume we should add a separate item for OSX and FreeBSD too?  We conditionally use TargetsWindows conditionally and want to be able to build for FreeBSD and OSX as well as Linux.  We don't conditionally compile anything specific for OSX or Linux, just Windows vs non-Windows.